### ISSUE TITLE
Error when installing Drutopia

### ISSUE THREAD

**Issue by cedewey** [drud/ddev#1108]
When trying to [install a fresh copy of Drutopia](https://gitlab.com/drutopia/documentation/blob/master/quickstart.md), I get the following error when running `ddev start`

`Failed to migrate db from bind-mounted db: failed to run migrate_file_to_volume.sh, err=container run failed with exit code 2 output=`

Currently running 

dba   	drud/phpmyadmin:v1.2.0
router	drud/ddev-router:v1.2.0
commit	v1.2.0
domain	ddev.local
cli   	v1.2.0
web   	drud/ddev-webserver:v1.2.0
db    	drud/ddev-dbserver:v1.2.0

on OSX 10.13.2

**Comment by rfay** [comment]
Hi @cedewey - you already have a project by that name existing; So it's trying to migrate it. I haven't been able to sort out the "exit 2" problem that's happened to some people, but for you, wanting to do a fresh install, use `ddev remove --remove-data --omit-snapshot` and you'll have it all solved.

**Comment by rfay** [comment]
I would actually love it if you could get me a tarball of your ~/.ddev/<projectname> and your project directory, if that were possible. It might give me a chance to debug that "error code 2" problem. You'd have to do this before doing the above.

**Comment by cedewey** [comment]
When I run `ddev remove --remove-data --omit-snapshot` I get the following error,  

`Failed to remove ddev project new-drutopia: failed to remove ddev project new-drutopia: could not find containers which matched search criteria: map[com.ddev.site-name:new-drutopia]`

I'll send a tarball via Slack. Thanks!

**Comment by rfay** [comment]
Yeah, after you send the tarballs just `rm -rf ~/.ddev/new-drutopia` and that will solve the problem as well. Thanks!
